src/api/project/project.test.ts

```diff
@@ -33,5 +39,18 @@ describe("project", () => {
       name: "sample-project",
       description: "sample-description",
     });
+
+    const getAllProjectResponse = await request
+      .get("/project");
+    expect(getAllProjectResponse.body).to.have.length(3);
+  });
+
+  it("can't create project because required parameter not given", async () => {
+    const createProjectResponse = await request
+      .post("/project")
+      .send({ name: "sample-project" });
+    expect(createProjectResponse.status).equal(500);
+    expect(createProjectResponse.body).to.has.property("isSuccess")
+      .equal("false");
```

Comment:
message nya jangan lupa di check juga. Coba bikin request failed terus contek error messagenya dan isi disini

```suggestion
    expect(createProjectResponse.body).to.deep.include({
       isSuccess: "false",
       message: "error message, check mongoose error message and fill here"
    });
```

Reply:
failed responsenya juga ada status code, apa ikut dimasukin kriteria juga status codenya?